sharkfuser/include/fusilli/attributes/tensor_attributes.h

```diff
@@ -257,6 +257,39 @@ getContiguousToChannelsLastPermuteOrder(size_t numDims) {
   return permuteOrder;
 }
 
+// Given a permutation vector, returns its inverse.
+inline std::vector<int64_t>
+inversePermutation(const std::vector<int64_t> &permutation) {
+  std::vector<int64_t> inverse(permutation.size());
+  for (size_t i = 0; i < permutation.size(); ++i) {
+    inverse[permutation[i]] = static_cast<int64_t>(i);
+  }
+  return inverse;
+}
+
+// Generates permute order to convert any strided layout to contiguous layout.
+// This computes the permutation needed to go from logical dims to physical dims
+// based on the stride.
+inline std::vector<int64_t>
+getToContiguousPermutation(const std::vector<int64_t> &stride) {
```

Comment:
Could you please double-check the function name? Looks like smth goes wrong here (not aligned with other functions at least)